### PR TITLE
[fixes #97892114] Change the generation of the EPCIS event id from MD5 hash to UUID random.

### PR DESCRIPTION
**Pull request by spacemonkeythe**

### CODE REVIEW

**Comment by jasperkremer** [comment]
It doesn't compile:
```
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /Users/jasper.kremer/Projects/retail-services-API/java/examples/src/main/java/com/nedap/retail/services/examples/WorkflowExample.java:[4,1] cannot find symbol
  symbol:   static generateEpcisEventID
  location: class
[ERROR] /Users/jasper.kremer/Projects/retail-services-API/java/examples/src/main/java/com/nedap/retail/services/examples/WorkflowExample.java:[42,29] cannot find symbol
  symbol:   method generateEpcisEventID(long,int)
  location: class com.nedap.retail.services.examples.WorkflowExample
[ERROR] /Users/jasper.kremer/Projects/retail-services-API/java/examples/src/main/java/com/nedap/retail/services/examples/WorkflowExample.java:[43,29] cannot find symbol
  symbol:   method generateEpcisEventID(long,int)
  location: class com.nedap.retail.services.examples.WorkflowExample
```

**Comment by spacemonkeythe** [comment]
I can't find the build on Jenkins for this repo, is that the way it should be?

File: java/examples/src/main/java/com/nedap/retail/services/examples/WorkflowExample.java [hunk 1]
```diff
@@ -39,8 +38,8 @@ public static void runExample(final Client client) {
             System.out.println(NEW_LINE + "Capturing some EPCIS events first...");
             final EpcisEventContainer epcisEventsContainer = new EpcisEventContainer();
             epcisEventsContainer.events = createEvents(locationId);
-            MESSAGE_IDS.add(generateEpcisEventID(ORGANIZATION_PREFIX, 1).toString());
-            MESSAGE_IDS.add(generateEpcisEventID(ORGANIZATION_PREFIX, 2).toString());
+            MESSAGE_IDS.add(UUID.randomUUID().toString());
```
Comment (jasperkremer) [inline java/examples/src/main/java/com/nedap/retail/services/examples/WorkflowExample.java]: It should not calculate new random UUID's.
Instead is should use the already calculated id's from `epcisEventsContainer.events`

**Comment by jasperkremer** [comment]
Merged as squashed commit.

**Pull request closed without merging**